pages/index.vue

```diff
@@ -138,6 +139,24 @@ export default defineComponent({
        * @hints use store and set loading true
        * @caution you have to assign new value to todos for it to update
        */
+      const headers = {
+        Authorization: 'Token ' + this.$store.getters.token,
+      }
+      this.$axios
+        .get('todo/', { headers })
+        .then(({ data }) => {
+          this.todos = []
+          if (data) {
+            for (let element of data) {
+              element.editing = false
+              this.todos.push(element)
+            }
+          }
+          console.log(this.todos)
+        })
+        .catch((e) => {
+          console.log(e)
```

Comment:
you can show toast instead of console logging

Reply:
Yes, Thank You!